src/elm/Page/Profile.elm

```diff
@@ -341,7 +341,7 @@ viewForm loggedIn isDisabled profile avatarS form =
                         , div [ class "input-counter" ]
```

Comment:
👍  on the refactor, but don't you think we should keep refactors away from new feature code as it makes it harder to review code and see changes?

Reply:
Why have them separated? what are the advantages? This in particular is a warning the elm-analyst emits in unused variables. no behavior or flow changes, i wouldn’t even call it a refactor 

When i think about refactor it’s more related to flow/logic improvements. here and in the other substitution of a case for an if there were nothing changed in the logic of the program